libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -351,8 +348,6 @@ const ParameterSection = ({
           editor,
           editorOptions,
           tokenEditor: true,
-          isTrigger,
-          isCallback: nodeType?.toLowerCase() === constants.NODE.TYPE.HTTP_WEBHOOK,
```

Comment:
By removing web hook check would we still show the listcallbackurl token?

Reply:
I have verified that we still do